packages/builder/package.json

```diff
@@ -58,7 +58,7 @@
     "@babel/preset-env": "^7.5.5",
     "@babel/runtime": "^7.5.5",
     "@rollup/plugin-alias": "^3.0.1",
-    "@sveltech/routify": "1.5.0-beta.16",
+    "@sveltech/routify": "1.5.0-beta.26",
```

Comment:
have the routify team any timescale around a non-beta release?

Reply:
Not entirely sure at the moment but it's supposed to be very  close to finished. I'll speak with the maintainers